source/gui/installerGui.py

```diff
@@ -340,6 +341,59 @@ def showInstallGui():
 	gui.mainFrame.postPopup()
 
 
+def _warnForNonEmptyDirectory(portableDirectory: str) -> bool:

```

Comment:
Given the function's purpose and return behaviour (returning a value as part of the prompt) consider naming the function something that reflects its dual nature:
```suggestion
def _warnAndConfirmForNonEmptyDirectory(portableDirectory: str) -> bool:

```